cf-oscore/src/main/java/org/eclipse/californium/oscore/Decryptor.java

```diff
@@ -140,7 +140,7 @@ protected static byte[] decryptAndDecode(Encrypt0Message enc, Message message, O
 			plaintext = enc.decrypt(key);
 
 		} catch (CoseException e) {
-			LOGGER.error(ErrorDescriptions.DECRYPTION_FAILED + " " + e.getMessage());
+			LOGGER.error(ErrorDescriptions.DECRYPTION_FAILED + " {}", e.getMessage());
 			throw new OSException(ErrorDescriptions.DECRYPTION_FAILED + " " + e.getMessage());
```

Comment:
In such a case, where the same message is used for logging and the Exception, I would got for:

```
String details = ErrorDescriptions.DECRYPTION_FAILED + " " + e.getMessage();
LOGGER.error("{}", details);
throw new OSException(details);
```

Reply:
Yes I realize searching around that this pattern happens very many times in the current code.

Is this good to do in general (also for fixed strings) or only when part of the message is gotten from some object?

Like 
```
LOGGER.error("{} {}", ErrorDescriptions.DECRYPTION_FAILED, e.getMessage());
throw new OSException(ErrorDescriptions.DECRYPTION_FAILED + " " + e.getMessage());
```
vs.
```
LOGGER.error(ErrorDescriptions.TOKEN_NULL);
throw new NullPointerException(ErrorDescriptions.TOKEN_NULL);
```

I could certainly go through and change this for all instances if that is better.